README.adoc

```diff
@@ -0,0 +1,177 @@
+= Eclipse uProtocol client vsomeip python
+:toc:
+
+Python uPClient implementation for SOME/IP using vsomeip
+
+== Overview
+
+Python module to leverage someip implementations developed from other programming languages.
+
+
+== Getting Started
+
+=== Prerequisites
+Download, Build, and Install vsomeip - [COVESA / vsomeip](https://github.com/COVESA/vsomeip)
+for windows support: [COVESA / vsomeip (fork)](https://github.com/justinlhudson/vsomeip)
+
+==== Linux
+1. clone vsomeip repo https://github.com/COVESA/vsomeip.
+2. Follow Build instructions for Linux on the GitHubPage: https://github.com/COVESA/vsomeip
+	a. sudo apt-get install asciidoc source-highlight doxygen graphviz
+	b. sudo apt-get install libboost-all-dev
+	c. sudo apt install cmake
+3. open a terminal window at vsomeip directory.
+4. mkdir build
+5. cd build
+6. cmake ..
+7. make
+8. sudo make install
+
+[!TIP]
+'error' loading configuration module
+sudo ldconfig
+
+==== Windows
+1. Install boost - https://sourceforge.net/projects/boost/files/boost-binaries/1.65.0/boost_1_65_0-msvc-14.1-64.exe/download
+2. Install Visual Studio (any edition) - With Development with C++ options: https://learn.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-170
+3. Clone the forked vsomeip repo https://github.com/justinlhudson/vsomeip
```

Comment:
same as above

Reply:
This is related to a Windows OS patch made by Justin for main C++ vsomeip Repo. He has made a PR to COVESA vsomeip repo. Once its merged, this can be updated, until then this remains